src/components/Sidebar.js

```diff
@@ -1,68 +1,81 @@
 import React, { Component } from "react";
 
 import AdminLink from "./AdminLink";
+import url from "../url";
 
 
+function SideBarLink(props) {
+  const {currentPath, name, params, children, className, ...otherProps} = props;
+  const targetUrl = url(name, params);
+  const active = currentPath === targetUrl ? "active" : "";
+  const classes = className !== undefined ? className : `list-group-item ${active}`;
```

Comment:
Nit: I'd rather prefer the `list-group-item` class to be explicitely set by calls when then need it.